### ISSUE TITLE
Table of contents for class methods

### ISSUE THREAD

**Issue by ivirshup** [python-graphblas/python-graphblas#541]
Hey,

This is a request about doc formatting. I think it would be helpful if I could see a table of contents for all the methods of a class when looking at the API docs.

Here's what the site currently looks like:

<img width="1661" alt="image" src="https://github.com/python-graphblas/python-graphblas/assets/8238804/7ecb0bf4-c179-4bfb-9d20-d545557fc618">

Here's an example of a table of contents structure containing methods

[link](https://docs.scvi-tools.org/en/stable/api/reference/scvi.model.LinearSCVI.html#scvi.model.LinearSCVI)

<img width="1661" alt="image" src="https://github.com/python-graphblas/python-graphblas/assets/8238804/34b03efa-fbb7-40ca-ae65-01c23afe5ae7">

**Comment by eriknw** [comment]
I agree 100%. That would be great, and looks like it's doable. Related: #410.

**Comment by eriknw** [comment]
Also, I really like small, achievable suggestions like this. We all know documentation needs improved, but it often seems like a large, daunting task. Please don't be shy about more suggestions :)

I'm also happy to chat with you anytime about sparse computing, 'cause I have thoughts
